src/index.js

```diff
@@ -1,19 +1,44 @@
-import React from "react";
+import React, { useState } from "react";
 import ReactDOM from "react-dom";
 import { BackButton } from "./shared/BackButton";
 import { TopHeader } from "./shared/TopHeader";
+import Chessboard from "chessboardjsx";
 
-const App = () => (
-  <div>
-    {/* This is just to test. Remove later */}
-    <TopHeader></TopHeader>
-    {/* This is just to test if prop is passing. We will change this our page instead of alert */}
-    <BackButton
-      onClick={() => {
-        alert("Yay, it works");
-      }}
-    />
-  </div>
-);
+const Header = (props) => <h1> {props.text} </h1>
+const Button = (props) => <button onClick={props.handleClick}> {props.text} </button>
+
+const Board = (props) => {
+  if (props.boardState === 'reset') {
+    return(<div> < Chessboard /> </div>)
+  }
+  else if (props.boardState === 'set') {
+    return(<div> < Chessboard position="start" /> </div>)
+  }
+  else {
+    return(<div> Error </div>)
+  }
+}
+
+const App = () => {
+  const [boardState, setBoardState] = useState('reset')
+
+  const resetBoard = () => {
+    setBoardState('reset')
+  }
+
+  const setBoard = () => {
+    setBoardState('set')
+  }
+
+
+  return(
+    <div>
+      < Header text="Social Connect - Play" />
```

Comment:
For Header and back button if you need it, use the shared components as we want the UI to be consistent?